src/pint/models/timing_model.py

```diff
@@ -228,6 +221,62 @@ def __init__(self, name="", components=[]):
         self.add_param_from_top(
             MJDParameter(name="FINISH", description="End MJD for fitting"), ""
         )
+        self.add_param_from_top(
+            strParameter(
+                name="INFO",
+                description="Tells TEMPO to write some extra information about frontend/backend combinations; -f is recommended",
```

Comment:
The argument to INFO could be any TOA flag, -f is what we use for fe/be combos.  But you could just specify -fe or even -gof or -snr.

Reply:
INFO will not be written to the output par file unless it is set, either in the input par file or on the model object, to some value. So the NANOGrav scripts will have to put INFO -f.